Update develop-client-apps.md

normesta opened the pull request:
Hello I'm a writer in Windows dev docs. I was asked to update this topic with the most recent information about different client dev technologies. Maira should review.

normesta committed:
Update develop-client-apps.md

I'm a writer in Windows dev docs. I was asked to update this topic.

dnfclas commented:
[![CLA assistant check](https://cla.dotnetfoundation.org/pull/badge/not_signed)](https://cla.dotnetfoundation.org/dotnet/docs?pullRequest=4088) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/docs?pullRequest=4088) before we can accept your contribution.<br/><br/>:x: normesta [sign now](https://cla.dotnetfoundation.org/dotnet/docs?pullRequest=4088)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/docs?pullRequest=4088) it.</sub>

mairaw commented on docs/framework/develop-client-apps.md:
```diff
@@ -29,7 +29,53 @@ ms.workload:
+
+:heavy_check_mark: Simple, clean, and trustworthy deployment to Windows 10 users by using the Desktop Bridge.
+
+Here's a few important trade-offs to consider. These applications run only on PCs and because we're making very low investments in these technologies, UIs in your application will appear out dated over time.
```
out dated -> outdated

mairaw commented on docs/framework/develop-client-apps.md:
```diff
@@ -29,7 +29,53 @@ ms.workload:
+
+:heavy_check_mark: Benefits from ongoing investment to the platform.
+
+Here's a few trade-offs to consider. These applications run only in Windows 10 and they run inside of a protected sandbox which means that file access is limited and communication outside of the application requires some work. The .NET APIs that you can use in a UWP app are limited, but this collection grows with each release. Because this is our investment going forward, we're addressing experie ...
```
missing comma before which

mairaw commented on docs/framework/develop-client-apps.md:
```diff
@@ -29,7 +29,53 @@ ms.workload:
+
+Here's a few important trade-offs to consider. These applications run only on PCs and because we're making very low investments in these technologies, UIs in your application will appear out dated over time.
+
+You can use some Windows 10 APIs directly from your app, but for many, you'll have to create a package by using the Desktop Bridge. To use APIs that depend on modern UIs, you'd have to go one step further and extend your solution with a separate UWP project so there's some work involved in using the most up-to-date Windows APIs.
```
you'll have -> you have

mairaw reviewed (changes requested):
Looks good @normesta. Left some comments for you to address.

mairaw commented:
Also tagging @Tanya-Solyanik and @merriemcgaw to review winforms messaging.

normesta committed:
Update develop-client-apps.md

Incorporating comments. Thanks Mairaw!

normesta commented:
Ok updated with your feedback. Thank you Maira!

Tanya-Solyanik commented on docs/framework/develop-client-apps.md:
```diff
@@ -73,9 +73,9 @@ Here's a few trade-offs to consider. These applications run only in Windows 10 a
 
 :heavy_check_mark: Simple, clean, and trustworthy deployment to Windows 10 users by using the Desktop Bridge.
```
@merriemcgaw - should we mention ClickOnce deployment as well since we mentioned win7 in the first point? ClickOnce deploys to all supported versions of windows.

merriemcgaw replied:
It does seem off not to have ClickOnce mentioned since deployment via Desktop Bridge is listed as a reason to use Windows apps.  You could simply add "and using ClickOnce to deploy to legacy Windows versions" or something to that effect.  It's reasonable to deemphasize ClickOnce on Win10 since we're trying to push customers towards Desktop Bridge/Store based deployments.  FWIW though ClickOnce is fully supported on Win10 and from what I've seen so far easier than using Desktop Bridge to deploy.

mairaw replied:
@normesta can you look at this feedback?

normesta committed:
Update develop-client-apps.md

I've been asked by Windows core PM's to keep positioning statements out of this doc until an upcoming (2 weeks or s) blog post about this exact conversation. I'll then update this doc to align with that positioning info.

normesta commented:
Hi all - I've been asked by our Windows Core PMs to exclude Windows Forms/WPF/UWP positioning information until they publish a blog post (about a week or so out) that will contain information about this exact subject. I'll issue a pull request that aligns with the messaging of that post.  For now, we just want to elevate the visibility of UWP in this page so I've tried to do that.

mairaw commented on docs/framework/develop-client-apps.md:
```diff
@@ -29,8 +29,14 @@ ms.workload:
 # Developing Client Applications with the .NET Framework
-There are multiple ways to develop Windows-based applications with the .NET framework that run locally on users' computers or devices. This section contains topics that describe how to create Windows-based applications by using Windows Presentation Foundation (WPF) or by using Windows Forms. However, you can also create web applications using the .NET Framework, and client applications for comput ...
-  
+There are several ways to develop Windows-based applications with the .NET framework. You can use any of these tools and frameworks: 
```
.NET framework -> .NET Framework

mairaw committed:
Update develop-client-apps.md

mairaw reviewed (approved):
After my last changes, it LGTM. Thanks @normesta. It should be live in the next day or so.
